homeassistant/components/sensor/sense.py

```diff
@@ -139,7 +139,13 @@ def icon(self):
 
     def update(self):
         """Get the latest data, update state."""
-        self.update_sensor()
+        from requests import ReadTimeout
```

Comment:
I don't think you should introduce exceptions from heretofore un-explicitly-used libraries. The `sense_energy` library should provide its own exception type and HASS should deal with _that_ instead.